### PR TITLE
Updated link Validation section

### LINKED ISSUE #210: (OUI Docs) Link
- [ ] Update instances of www.elastic.co link to https://Opensearch.org
- [ ] Update instances of https://github.com/elastic/eui to https://github.com/opensearch-project/oui

Section: Link validation
- [ ] Replace //elastic.co with //opensearch.org

### PR DESCRIPTION
**Pull request by AbhishekReddy1127**
Signed-off-by: AbhishekReddy1127 <nallamsa@amazon.com>

### Description
Updated the links to the Opensearch website in the Links section.

Screenshot 2023-02-07 at 2 05 23 PM
Screen Shot 2023-02-02 at 11 01 18 PM

 
### Issues Resolved
#210 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by KrooshalUX — CHANGES_REQUESTED** [review]
Correct Opensearch to OpenSearch

File: src-docs/src/views/link/link.js [hunk 1]
```diff
@@ -17,15 +17,15 @@ export default () => (
   <OuiText>
     <p>
       Open the{' '}
-      <OuiLink href="http://www.elastic.co" target="_blank">
-        Elastic website
+      <OuiLink href=" https://Opensearch.org" target="_blank">
```
Comment (BSFishy) [inline src-docs/src/views/link/link.js:20]: Could these be changed to not have a space before the url?

File: src-docs/src/views/link/link.js [hunk 2]
```diff
@@ -17,15 +17,15 @@ export default () => (
...
+        OpenSearch website
       </OuiLink>{' '}
       in a new tab. Setting <OuiCode>target=&ldquo;_blank&rdquo;</OuiCode> also
       defaults to <OuiCode>{'external={true}'}</OuiCode>.
     </p>
     <p>
       This{' '}
-      <OuiLink href="http://www.elastic.co" external>
+      <OuiLink href=" https://Opensearch.org" external>
```
Comment (BSFishy) [inline src-docs/src/views/link/link.js:28]: See previous comment

**Review by BSFishy — CHANGES_REQUESTED** [review]

**Review by BSFishy — APPROVED** [review]

**Comment by KrooshalUX** [comment]
One note - I am unable to link this to issue 210 using the "development" feature in github, not sure why - calling out to make sure we close the appropriate issue on merge.

**Pull request merged**